Tests/TestingApps/PurchaseTesterSwiftUI/Shared/PurchaseTesterApp.swift

```diff
@@ -12,69 +12,41 @@ import RevenueCat
 @main
 struct PurchaseTesterApp: App {
 
-    private let delegate = Delegate()
-
-    init() {
-        let configuration: Configuration =
-            .builder(withAPIKey: Constants.apiKey)
-            .with(usesStoreKit2IfAvailable: true)
-            .build()
-
-        if let proxyURL = Constants.proxyURL {
-            Purchases.proxyURL = URL(string: proxyURL)!
-        }
-
-        Purchases.logLevel = .debug
-        Purchases.configure(with: configuration)
-        Purchases.shared.delegate = self.delegate
-    }
-
-    @State private var revenueCatCustomerData = RevenueCatCustomerData()
+    @State
+    private var configuration: ConfiguredPurchases?
     
     var body: some Scene {
         WindowGroup {
-            ContentView()
-                .environmentObject(revenueCatCustomerData)
-                .task {
-                    for await customerInfo in Purchases.shared.customerInfoStream {
-                        self.revenueCatCustomerData.customerInfo = customerInfo
-                        self.revenueCatCustomerData.appUserID = Purchases.shared.appUserID
+            NavigationView {
+                if let configuration {
+                    ContentView(configuration: configuration)
+                        .toolbar {
+                            ToolbarItem(placement: .navigationBarLeading) {
+                                Button {
+                                    self.configuration = nil
```

Comment:
Does the iOS SDK keep some sort of shared instance of `Purchases`? Just wondering if this is enough to reset it.

Reply:
Technically not, but it will be re-created when it gets re-configured.